xprof/btx_aggreg_params.yaml

```diff
@@ -1,14 +1,4 @@
 :type: map
 :entries:
-  - :name: display
-    :type: string
-  - :name: name
-    :type: string
-  - :name: display_mode
-    :type: string
-  - :name: display_metadata
-    :type: bool
-  - :name: display_name_max_size
-    :type: integer_unsigned
-  - :name: display_kernel_verbose
```

Comment:
Why are these all disappearing?

Reply:
They where never used, I guess copy/past error from the tally component